examples/07-EMIT/ComputeProtectionLevels.py

```diff
@@ -18,299 +18,299 @@
 import pyaedt
```

Comment:
In the future, can we put a version check at the top and exit if the version requirement isn't met, instead of commenting the whole thing? It will get hard to review and work with the diffs if we have to comment/uncomment. We could use the block comments (""") if needed, but maybe a version check would work?

Reply:
Yea, that probably makes sense. I'm going to make that a separate DE, I just uncommented things here to do some basic testing and ensure things still work.